exonum-java-binding/fakes/pom.xml

```diff
@@ -21,6 +21,12 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-java-app</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
     <dependency>
       <groupId>com.exonum.binding</groupId>
       <artifactId>exonum-java-binding-core</artifactId>
```

Comment:
Core can be removed such as it comes from the app as transitive?

Reply:
Thanks, probably yes, we can rely on app having core.